css/css-ui-3/cursor-auto-007.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<title>CSS Basic User Interface Test: cursor:auto on unselectable text</title>
+<link rel="author" title="Florian Rivoal" href="https://florian.rivoal.net/">
+<link rel="help" href="http://www.w3.org/TR/css3-ui/#cursor">
+<link rel="help" href="https://drafts.csswg.org/css-ui-4/#content-selection">
+<meta name="flags" content="interact">
+<meta charset="UTF-8">
+<meta name="assert" content="The 'auto' cursor value does the same as 'default' over unselectable text.">
+<style>
+#test {
+  border: solid blue;
+  width: 200px;
+  cursor: auto;
+}
+#test * {
+  cursor: url("support/cursors/fail.png"), help !important; /* Override UA styles, regardless of specificity */
+  cursor: auto !important; /* Override UA styles, regardless of specificity */
+}
+#ref {
+  width: 100px;
+  height: 100px;
+  cursor: default;
+  border: solid orange;
+}
+:root {
+  cursor: help; /* give the root element a different cursor so that
+  it is easy to tell if something changes when hovering the target.*/
+}
+
+/* Hide the text if we cannot make it unselectable.
+ user-select is not part of css-ui level 3, so we should not depend on it
+ but it is nice to test it if it is supported.
+ Test for level 4 can take the conditional out. */
```

Comment:
Ok, I guess this explains that the link to `css-ui-4` was done in purpose. :wink:

Reply:
Right.